### ISSUE TITLE
Add html Viewer

### ISSUE THREAD

**Issue by kieranjmartin** [gowerc/diffdf#8]

**Comment by gowerc** [comment]
I guess we should discuss what exactly we want from this.

Do we wish for differences to be displayed in the HTML viewer by default or just that we have an export to html option ?  I am leaning towards displaying HTML by default as it is prettier. 

We would then though need to be dependent on the rstudioapi  as trying to browse and open a html file without the rstudio being open  ( rstudioapi::isAvailable() )  it would be a bit silly to open the html file.  

i.e. if rstudio active  load html else print ascii

**Comment by kieranjmartin** [comment]
Yes I think that would be ideal, printing would open up the nice html file in the viewer which you could then open in a new page. I mean if we wanted to really show off we could make a little shiny app to help you explore....

**Comment by gowerc** [comment]
lol;  lets try walking before we run :)

**Comment by gowerc** [comment]
Just to say that I've made a start on this in the branch `render_html`

**Comment by gowerc** [comment]
Making some good progress with this now though a lot more work is required !!!

![image](https://user-images.githubusercontent.com/8447209/79009192-dccc4580-7b56-11ea-9594-4676b260a392.png)
